src/webviews/extension-side/dataviewer/dataViewerCommandRegistry.ts

```diff
@@ -99,7 +99,10 @@ export class DataViewerCommandRegistry implements IExtensionSingleActivationServ
                         this.debugService.activeDebugSession.configuration
                     );
 
-                    pythonEnv && (await this.dataViewerDependencyService.checkAndInstallMissingDependencies(pythonEnv));
+                    pythonEnv &&
+                        (await this.dataViewerDependencyService.checkAndInstallMissingDependencies({
+                            interpreter: pythonEnv
+                        }));
```

Comment:
The interpreter is only being used in Node.js, in the web the interpreter is ignored and the debugger is used (in this case, because here we don't have a kernel attached).